src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindow.kt

```diff
@@ -1,22 +1,13 @@
 package com.github.mitchellolsthoorn.testgenie.toolwindow
 
 import com.intellij.openapi.ui.Messages
-import com.intellij.openapi.wm.ToolWindow
-import java.awt.Dimension
 import java.awt.event.ActionEvent
-import javax.swing.JButton
-import javax.swing.JComboBox
-import javax.swing.JComponent
-import javax.swing.JLabel
-import javax.swing.JPanel
-import javax.swing.JTextField
+import javax.swing.*
```

Comment:
Is a `*` import necessary?

Reply:
It was automatically inserted by IntelliJ and we don't have any rule that forbids `*` imports, so I think it should be fine